cloud/deploy/deploy.go

```diff
@@ -426,40 +433,40 @@ func getDeploymentInfo(deploymentID, wsID, deploymentName string, prompt bool, c
 
 	// check if deploymentID or if force prompt was requested was given by user
 	if deploymentID == "" || prompt {
-		currentDeployment, err := deployment.GetDeployment(wsID, deploymentID, deploymentName, false, client, coreClient)
+		currentDeployment, err := deployment.GetDeployment(wsID, deploymentID, deploymentName, false, corePlatformClient, coreClient)
 		if err != nil {
 			return deploymentInfo{}, err
 		}
-		coreDeployment, err := deployment.CoreGetDeployment(currentDeployment.Workspace.ID, currentDeployment.Workspace.OrganizationID, currentDeployment.ID, coreClient)
+		coreDeployment, err := deployment.CoreGetDeployment(currentDeployment.OrganizationId, currentDeployment.Id, corePlatformClient)
 		if err != nil {
 			return deploymentInfo{}, err
 		}
 		var desiredDagTarballVersion string
-		if coreDeployment.DesiredDagTarballVersion != nil {
-			desiredDagTarballVersion = *coreDeployment.DesiredDagTarballVersion
+		if coreDeployment.DagTarballVersion != nil {
```

Comment:
Why is this check changing?

Reply:
desiredDagTarballVersion does not seem to exist in v1beta1 so I had to change it to dagTarballVersion

Reply:
I think we need to update v1beta1 endpoint to expose that info.